wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/SyncMLSOneToOneConversationActionHandler.swift

```diff
@@ -114,6 +114,34 @@ final class SyncMLSOneToOneConversationActionHandler: ActionHandler<SyncMLSOneTo
             action.fail(with: .unknown(status: errorInfo.status, label: errorInfo.label, message: errorInfo.message))
         }
     }
+
+    private func updateOrCreateConversation(
+        action: SyncMLSOneToOneConversationAction,
+        payload: Payload.Conversation,
+        publicKeys: BackendMLSPublicKeys? = nil
+    ) {
+        Task { [action] in
```

Comment:
I'd prefer to remove this line and declare the func async.
Then on the calling side it's obvious that it contains async code which hasn't completed when this call returns.

Reply:
This would require refactoring the `ActionHandler` class, which I would not do in this PR.